config/etc/system-image/writable-paths

```diff
@@ -13,6 +13,7 @@
 /tmp                                    none                    temporary   none        defaults
 /mnt                                    none                    temporary   none        defaults
 /media                                  none                    temporary   none        defaults
+/boot/androidboot                       auto                    persistent  none        none
```

Comment:
the configuration is inside the boot.img and can be read at any time by "abootimg -i" directly from the boot partition or for a boot.img file, there is no need to have it somewhere separate. also, all info about the currently running configuration of snap_core or snap_kernel should be in /proc/cmdline